code/obj/submachine/seed.dm

```diff
@@ -870,10 +868,8 @@
 				if (!I)
 					return
 				if (I == src.extract_to) src.extract_to = null
-				TRANSFER_OR_DROP(src, I)
+				TRANSFER_OR_DROP(src, I) // causes Exited proc to be called
 				usr.put_in_hand_or_eject(I)
-				src.inserted = null
-				. = TRUE
```

Comment:
Did you accidentally remove line 876?

Reply:
No, `Exited` should cause a UI update, which is what the return for `ui_act` is normally for.